chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/aux/MaxLoadThreads.java

```diff
@@ -2,9 +2,8 @@
 
 /**
  * The maximum number of threads, which Jagger engine can create to provide the requested load.
- * <p>
- * Created by Andrey Badaev
- * Date: 16/11/16
+ * By default it equals 4000. You can change this value in property file
```

Comment:
It is not clear: 4000 is maximum allowed value? Or if it is exactly 'Default' value than we need appropriate constructor in JLoadProfileRps